src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/BehandlingService.kt

```diff
@@ -43,6 +43,16 @@ class BehandlingService(
         .filter { !it.erHenlagt() && it.status == BehandlingStatus.AVSLUTTET }
         .maxByOrNull { it.opprettetTidspunkt }
 
+    /**
+     * Henter siste iverksatte behandling FØR en gitt behandling.
```

Comment:
Før eller for?

Reply:
Vi bruker ikke denne metoden i behandlingsresultat

Reply:
Fjernet metoden 👍